assembl/static2/js/app/components/common/tree.jsx

```diff
@@ -210,9 +199,9 @@ class Tree extends React.Component {
     // Reset the global prevStopIndex to not overfetch posts when changing idea
     // or to avoid recreating all dom nodes if we go back to the same idea.
     this.cache.clearAll();
-    prevStopIndex = 0;
+    this.prevStopIndex = 0;
 
-    document.addEventListener('rowHeightRecomputed', this.nuggetsManager.update);
+    if (this.rootRef) this.rootRef.addEventListener('rowHeightRecomputed', this.nuggetsManager.update);
```

Comment:
This part was a bit confusing to me. But I understand now, you don't want a higher level DOM object consuming events from all columns. Each column throws their own event.

More importantly, we *should* pass a prop to this component that enables/disables nugget management on the tree. In case of columns, there will never be nuggets. Therefore you're adding event listener and throwing events for no gain. The component can be more efficient.